test/common/http/conn_manager_impl_fuzz_test.cc

```diff
@@ -482,15 +482,15 @@ class FuzzStream {
             Fuzz::fromHeaders<TestResponseHeaderMapImpl>(response_action.headers()));
         // The client codec will ensure we always have a valid :status.
         // Similarly, local replies should always contain this.
-        uint64_t status;
-        try {
-          status = Utility::getResponseStatus(*headers);
-        } catch (const CodecClientException&) {
+        const auto status = Utility::getResponseStatusOrNullopt(*headers);
+        if (!status.has_value()) {
           headers->setReferenceKey(Headers::get().Status, "200");
         }
         // The only 1xx header that may be provided to encodeHeaders() is a 101 upgrade,
         // guaranteed by the codec parsers. See include/envoy/http/filter.h.
-        if (CodeUtility::is1xx(status) && status != enumToInt(Http::Code::SwitchingProtocols)) {
+        if (CodeUtility::is1xx(status.value_or(enumToInt(Http::Code::OK))) &&
+            status.value_or(enumToInt(Http::Code::OK)) !=
```

Comment:
The `value_or` here is redundant. Once you check `has_value` above, we can do it without `value_or` by small code tweaks. 

But please address my question above first and then we can change here accordingly.

Reply:
Yes. exactly.  Thanks！